af/fastpath/fastpath/s3feeder.py

```diff
@@ -80,25 +82,85 @@ def load_multiple(fn: str) -> Generator[MsmtTup, None, None]:
             #     metrics.incr("yaml_normalization")
             #     yield (None, msm)
 
+    elif fn.endswith(".tar.gz"):
+        # minican with missing gzipping :(
```

Comment:
I'm not sure I understand this comment. Are you saying that minicans don't have a gzip layer on top of them, but yet they still have the gzip extension?

Reply:
(Yes, as discussed)